doc/nrf/protocols/thread/device_types.rst

```diff
@@ -0,0 +1,85 @@
+An important difference between FTDs and MTDs is that FTDs subscribe to special "all-routers" multicast addresses (``ff02::2`` and ``ff03::2``, see `IPv6 multicast addressing`_).
+Because of these reasons, they typically consume more power than MTDs.
+
+Full Thread Devices can be further divided into three categories:
+
+* Router
+* Router Eligible End Device (REED)
+* Full End Device (FED)
+
+Routers can be regarded as the backbone of a Thread network.
+They maintain the routing table and forward messages to other devices.
+A router is required for a Minimal End Device to join the network.
+Routers, unlike End Devices, can also be used to extend the network range.
+The maximum number of routers in a single Thread network is 32.
+
+Both Routers and REEDs have routing capabilities, while FEDs do not.
+REEDs function similar to FEDs, but they can become Routers if needed.
+If a REED is the only device in range of an End Device trying to join the network it will promote itself to a Router (see `Joining an existing Thread network`_ in the OpenThread documentation for more information).
+Conversely, when a Router has no children it can downgrade itself to a REED.
+
+FEDs don't require a parent to function, but they can't become parents (Routers) themselves.
+
+.. _thread_types_mtd:
+
+Minimal Thread Devices
+**********************
+
+Minimal Thread Devices (MTDs) are devices that do not maintain a routing table and are typically low-power devices that are not always on.
+They can only be End Devices, and they always need a parent to function.
+They forward all their messages to their parent.
+
+Minimal Thread Devices can be further divided into three categories:
+
+* Minimal End Device (MED)
+* Sleepy End Device (SED)
+* Synchronized Sleepy End Device (SSED)
+
+MEDs are the most basic MTDs, and their radio is always on.
+They operate like FEDs, but they don't keep a routing table.
+
+SEDs try to limit their power consumption by sleeping most of the time, waking up periodically to poll for messages from their parent.
+After waking up, they send a data request to their parent.
+If the parent has any pending messages, it will send them to the SED.
+Otherwise, the parent will send a response indicating no pending messages.
+
+SSEDs operate similarly to Sleepy End Devices, but they are synchronized with their parent.
+They wake up at the same time as their parent, eliminating the need for polling for messages.
+If the parent has messages for the SSED, it sends them during the designated transmission window and the SSED allows the transmission to finish if radio activity is detected during the transmission window.
+Conversely, if there is no radio activity during the specified transmission window duration (indicating the parent has no messages for the SSED), the SSED returns to sleep.
+This synchronization results in lower power consumption, primarily because the SSED doesn't need to poll for messages, keeping transmission windows short.
```

Comment:
```suggestion
The SSED synchronization results in lower power consumption compared with the SED, primarily because the SSED doesn't need to poll for messages, keeping transmission windows short.
```